app/controllers/hiring_staff/sign_in/dfe/sessions_controller.rb

```diff
@@ -116,7 +116,7 @@ def organisation_id_present?
   end
 
   def allowed_user?
-    school_urn.present? || trust_uid.present? ||
-      (local_authority_code.present? && ALLOWED_LOCAL_AUTHORITIES.include?(local_authority_code))
+    school_urn.present? || trust_uid.present? || (local_authority_code.present? &&
+      (Rails.env.staging? || Rails.env.development? || ALLOWED_LOCAL_AUTHORITIES.include?(local_authority_code)))
```

Comment:
I forget what environment the review apps identify as. Are they included here?

Reply:
Staging